include/dsAudio.h

```diff
@@ -211,52 +211,6 @@ dsError_t  dsAudioPortTerm();
  */
 dsError_t  dsGetAudioPort(dsAudioPortType_t type, int index, intptr_t *handle);
```

Comment:
dsGetAudioFormat - is this function returns the current format of playback stream or the format output to the port?